src/components/templates/Message/index.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import PropTypes from 'prop-types'
+import { ustState,useEffect } from 'react'
+
+const Message = ({ style, className, children }) => {
+    // const onChange = e => {
+    //     console.log(e.target.value)
+    // }
+    const checkMessageLength = () => {
+        // const [messageHeight , setHeight] = useState(40)
+        const messageLength = children.length
+        const messageHeight = (messageLength / 12) * 40
+        style.height = `${40 + messageHeight}px`
+    }
+    useEffect(() => {
+        checkMessageLength()
+    })
+    return (
+        <MessageBox style={style} className={className}>
+            {children}
+        </MessageBox>
+    )
+    
+}
+
+
+
+Message.propTypes = {
+    style: PropTypes.objectOf(PropTypes.string),
+    className: PropTypes.string,
+    children: PropTypes.string,
+}
+
+
+
+const MessageBox = styled.div`
+    width: 180px;
+    height: 100%;
+    color:#000000;
+    font-size:14px;
+    background-color:#EEEEEE;
+    padding-top:10px;
+    padding-bottom:10px;
+    padding-left:19px;
+    padding-right:19px;
```

Comment:
```suggestion
padding: 10px 19px;
```
상하, 좌우가 같은 경우 축약하여 적어줄 수 있습니다! 😊